SECURITY.md

```diff
@@ -261,11 +261,31 @@ distribution should:
       marketing copy, etc.) that it is built on top of Envoy. E.g.,
       "SuperAwesomeCloudProvider's Envoy as a Service (EaaS)". An infrastructure
       service that uses Envoy for a product but does not publicly say they are
-      using Envoy does not qualify. This is essentially IaaS or PaaS, if you use
-      Envoy to support a SaaS, e.g. "SuperAwesomeCatVideoService", this does not
-      qualify.
-2. Have a user or customer base not limited to your own organization. We will use the size
-   of the user or customer base as part of the criteria to determine
+      using Envoy does not *generally* qualify (see option 3 that follows). This is essentially IaaS
+      or PaaS. If you use Envoy to support a SaaS, e.g. "SuperAwesomeCatVideoService", this does not
+      *generally* qualify.
+
+   OR
```

Comment:
Should we set an upper bound on the number of companies we are willing to admit before doing some sort of review of this process? I feel that we can't reasonably expect secrecy if there are more than a couple of dozen on the list, and there is at least that many companies today that basically meet the "well known" criteria.

What if medium sized company X is admitted, but then massive global company Y wants to be added but we have way too many folks on there already? Is there a criteria to eject?

Reply:
Good question. Let me think about this. Curious to hear the thoughts of others.

Reply:
We do say "*limited* set of end users" above. So I think we should set an upper bound. How about 30?

Reply:
30 seems a bit high to me. Maybe 10? Thoughts?